mount-tmpfs

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# This script simply mounts a tmpfs filesystem at build/test/cassandra
```

Comment:
please add information how is this script supposed to be used.
Is it used automatically by the build system?
Is one expected to run it manually? When?